docs/tables/aws_lambda_function.md

```diff
@@ -104,3 +104,16 @@ from
 where
   dead_letter_config_target_arn is null;
 ```
+
+
+### List runtime settings for each lambda function
```

Comment:
```suggestion
### List runtime settings for each function
```

We should not include the service name in the description unless we need to differentiate between services